src/relay/index.ts

```diff
@@ -0,0 +1,40 @@
+import 'isomorphic-fetch';
+import {StandardRelayerApi, SignedOrder} from '../types';
+
+export class Relay {
+    private url: string;
+    private version: number;
+    constructor(url: string) {
+        this.url = url;
+        this.version = 0;
+    }
+    public async getTokenPairsAsync(): Promise<StandardRelayerApi.RelayerApiTokenTradeInfo[]> {
+        const tokenPairs = await this._requestAsync('token_pairs', 'GET');
```

Comment:
I'd prefer we add the `/` at the beginning of the supplied path. It makes it very obvious that it's a path.